plugins/schedule/base/lib/model/ScheduleEvent.php

```diff
@@ -350,6 +350,11 @@ public function getTemplateEntryId()
 		return null;
 	}
 
+	public function getSourceEntryId()

```

Comment:
why it's not like
plugins/schedule/base/lib/model/LiveStreamScheduleEvent.php:30
also, it wont conflict with the existing one defined here?
alpha/apps/kaltura/lib/scheduleEvent/ILiveStreamScheduleEvent.php:5

Reply:
Its like this as the index contains more than 1 type of ScheduleEvent and not all of them have the source entryId